src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        Random random = new Random();

```

Comment:
random should be instantiated once and reused multiple times

consider making it class level field
```suggestion
        Random random = new Random();

```